src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -3165,25 +3190,38 @@ void AwtWindow::_ModalEnable(void *param)
         AwtWindow::SetModalBlocker(windowHWnd, NULL);
     }
 
-  ret:
+
```

Comment:
I recommend deleting this line, since it does not make sense to have two consecutive blank lines here.